notebooks/238-dolly-2-instruction-following/238-dolly-2-instruction-following.ipynb

```diff
@@ -0,0 +1,483 @@
+{
```

Comment:
<p>I think comparison with chatGPT in very beginning is not entirely valid, but can work. chatGPT is considering context and previous conversation (via roles), here it is simple prompt-answer model.</p><p>Proper comparison is probably InstructGPT indeed.</p><p> </p><p>I would suggest to mention where we will be able to run this model using OpenVINO. it will be possible on CPU and iGPU and even dGPU and on plain laptop without a need for expensive GPU to try.</p><p> </p><p>HF Optimum -&gt; HF Optimum-intel, Those are different libraries I believe.But maybe difference is not major...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1041/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>